medcat/meta_cat.py

```diff
@@ -14,6 +14,10 @@
 class MetaCAT(object):
     r''' TODO: Add documentation
     '''
+
+    # Custom pipeline component name
+    name = 'meta_cat'
```

Comment:
Does this make sense since all meta_cat objects usually are named by the function, e.g. negation, experiencer etc. So based on the category name